xbmc/music/infoscanner/MusicInfoScanner.cpp

```diff
@@ -42,7 +42,6 @@
 #include "settings/Settings.h"
 #include "FileItem.h"
 #include "guilib/LocalizeStrings.h"
-#include "utils/StringUtils.h"
```

Comment:
@Montellese please take a look at #6518 , it removes some occurrences where this was the case